### PR TITLE
[ALLUXIO-2213]Add doc for Running-HBase-on-Alluxio

### PR DESCRIPTION
**Pull request by TeddyBear1314**
https://alluxio.atlassian.net/browse/ALLUXIO-2213

I have created a document for Running HBase on Alluxio.
Welcome for your comments: @apc999 @yupeng9

### CODE REVIEW

File: docs/en/Running-HBase-on-Alluxio.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+</property>
+<property>
+  <name>hbase.rootdir</name>
+  <value>alluxio://<hostname>:<port>/hbase</value>
+</property>
+```
+
+# Distribute the Alluxio Client jar
+
+We need to make the Alluxio `jar` file available to HBase, because it contains the configured
```
Comment (yupeng9) [inline docs/en/Running-HBase-on-Alluxio.md]: To be more specific: Alluxio client `jar`

File: docs/en/Running-HBase-on-Alluxio.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+Tips:You do not need to create the /hbase directory in Alluxio, HBase will do this for you.
+
+```xml
+<property>
+  <name>fs.alluxio.impl</name>
+  <value>alluxio.hadoop.FileSystem</value>
+</property>
+<property>
+  <name>fs.alluxio-ft.impl</name>
+  <value>alluxio.hadoop.FaultTolerantFileSystem</value>
```
Comment (yupeng9) [inline docs/en/Running-HBase-on-Alluxio.md]: Could you also document how to deploy hbase with Alluxio in HA mode?

File: docs/en/Running-HBase-on-Alluxio.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+[Apache HBase Configuration](https://hbase.apache.org/book.html#configuration).
+
+# Configuration
+
+Apache HBase allows you to use Alluxio through a generic file system wrapper for the Hadoop file system.
+Therefore, the configuration of Alluxio is done mostly in HBase configuration files.
+
+#### Set property in `hbase-site.xml`
+
+You need to add the following three properties to `hbase-site.xml` in your HBase installation `conf` directory:
```
Comment (yupeng9) [inline docs/en/Running-HBase-on-Alluxio.md]: Should it be set for all the Hbase nodes?

**Comment by yupeng9** [comment]
Thanks @TeddyBear1314 for adding these useful instructions! I left some comments.

File: docs/en/Running-HBase-on-Alluxio.md (hunk 1 above)
Comment (TeddyBear1314) [inline docs/en/Running-HBase-on-Alluxio.md]: I agree with that! Thanks for the comments!

File: docs/en/Running-HBase-on-Alluxio.md (hunk 3 above)
Comment (TeddyBear1314) [inline docs/en/Running-HBase-on-Alluxio.md]: yes, you are right! I'll add some instructions!

File: docs/en/Running-HBase-on-Alluxio.md (hunk 2 above)
Comment (TeddyBear1314) [inline docs/en/Running-HBase-on-Alluxio.md]: ok, I am working on it! how do I start Alluxio when I finshed configuring the HA mode and how to test if it's working?

Comment (TeddyBear1314) [inline docs/en/Running-HBase-on-Alluxio.md]: ok, I have fix the problem, just wait for a few minutes before a standby master becomes active!!

File: docs/en/Running-HBase-on-Alluxio.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+# Distribute the Alluxio Client jar
+
+We need to make the Alluxio `jar` file available to HBase, because it contains the configured
+`alluxio.hadoop.FileSystem` class.
+
+There are two ways to achieve that:
+- Put the `alluxio-core-client-{{site.ALLUXIO_RELEASED_VERSION}}-jar-with-dependencies.jar` file into the
+  `lib` directory of HBase.
+- Specify the location of the jar file in the `HBASE_CLASSPATH` environment variable (make sure it's available
+on all cluster nodes). For example like this:
```
Comment (apc999) [inline docs/en/Running-HBase-on-Alluxio.md]: NIT: "For example", removing "like this" which is redundant.

**Comment by TeddyBear1314** [comment]
the HA mode can cause RegionServer go down when the standby master becomes active which costs about 2 min on my machine, I think I can add this part when I fix the "Regionserver go down" problem.

**Comment by apc999** [comment]
+1 LGTM. thanks for adding the awesome guide and help people understand Alluxio ecosystem better.

**Comment by yupeng9** [comment]
LGTM. @TeddyBear1314 do you plan to include the HA part in this PR, or a future one? Either one is fine to me.

**Comment by apc999** [comment]
I propose to have it in a separate PR (so people could have a guideline of HBase earlier from alluxio.org). 
is it ok @yupeng9 ?

**Comment by yupeng9** [comment]
SGTM

**Comment by TeddyBear1314** [comment]
ok, I'll pay attention to the "HBase on Alluxio HA mode" and add the HA part when I figure out a proper solution! Thank you for the comments @yupeng9 @apc999.

**Pull request merged**